src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -703,9 +704,10 @@ public int hashCode()
     @Override
     public String toString()
     {
-        return author != null
-            ? String.format("M:%#s:%.20s(%s)", author, this, getId())
-            : String.format("M:%.20s", this); // this message was made using MessageBuilder
+        return new EntityString(this)
+                .addMetadata("author", author.getAsTag())
+                .addMetadata("content", String.format("%.20s ...", this))
+                .toString();
```

Comment:
Are we dropping the author null check here because ReceivedMessage can't come from MessageBuilder anymore?

Reply:
Sounds good.